src/pages/trips/Trips.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from 'react';
+import { Nav, NavItem, NavLink, Container, Col, Row } from 'reactstrap';
+import { Query, compose, graphql } from 'react-apollo';
+import { Route, Redirect, Switch } from 'react-router';
+import { NavLink as RRNavLink } from 'react-router-dom';
+
+import AudioLoading from 'shared/loading/AudioLoading';
+import NotFound from 'components/routes/NotFound';
+import { GET_GUEST_SORTED_BOOKINGS, Booking, Currency, GUEST_CANCEL_BOOKING } from 'networking/bookings';
+import { differenceInDays } from 'date-fns';
+import { cancel, loadWeb3 } from 'utils/web3';
+
+const Trips = () => {
+  return (
+    <Query query={GET_GUEST_SORTED_BOOKINGS}>
+      {({ loading, error, data }) => {
+        if (loading) {
+          return <AudioLoading height={48} width={96} />;
+        }
+        if (error || !data) {
+          return <h1>{error ? error.message : 'Error / No Data'}</h1>;
+        }
+
+        const { cancelled, current, past, upcoming } = data;
+        const isCurrentEmpty = !(current || []).length;
+        // const isUpcomingEmpty = !(upcoming || []).length;
+        return (
+          <Container>
+            <h1>Trips</h1>
+            <hr />
+            <Nav tabs>
+              {[
+                {
+                  tag: RRNavLink,
+                  to: '/work/trips/current',
+                  title: 'Current',
+                },
+                {
+                  tag: RRNavLink,
+                  to: '/work/trips/upcoming',
+                  title: 'Upcoming',
+                },
+                {
+                  tag: RRNavLink,
+                  to: '/work/trips/past',
+                  title: 'Past',
+                },
+                {
+                  tag: RRNavLink,
+                  to: '/work/trips/cancelled',
+                  title: 'Cancelled',
+                },
+              ].map(({ title, tag, to }) => (
+                <NavItem key={to}>
+                  <NavLink tag={tag} to={to}>
```

Comment:
i think you can remove the `tag` attribute and just put `<RRNavLink />` as the tag

Reply:
good catch! thx